### ISSUE TITLE
2D Export of Arcs

### ISSUE THREAD

**Issue by gumyr** [gumyr/build123d#285]
There appears to be a problem with the export of 2D arcs as shown below:
```py
from build123d import *
from ocp_vscode import *

cylinder = Plane.XZ * Cylinder(10, 50)

visible, hidden = cylinder.project_to_viewport((100, -100, 100))
show_object(visible, name="v")
show_object(hidden, name="h")
exporter = ExportSVG(scale=1)
exporter.add_layer("Visible")
exporter.add_layer("Hidden", line_color=(99, 99, 99), line_type=LineType.ISO_DOT)
exporter.add_shape(visible, layer="Visible")
exporter.add_shape(hidden, layer="Hidden")
exporter.write(f"test_circle.svg")
```
![image](https://github.com/gumyr/build123d/assets/85638518/a5d77cc4-ff6c-4a0c-b8a0-0836a8549dee)
![image](https://github.com/gumyr/build123d/assets/85638518/65fc4560-e959-42ee-a855-64b51a78fb88)

**Comment by jrmobley** [comment]
Calculation of the ellipse rotation was reversed.  Fixed by d62f898.

**Comment by jrmobley** [comment]
Pics, or it didn't happen.
<img width="555" alt="fixed-ellipses" src="https://github.com/gumyr/build123d/assets/2137108/15c07b1e-70f4-4228-997d-fa38a4daf79c">
